src/challenges/challengeOne/3.Table/Table.test.tsx

```diff
@@ -1,8 +1,36 @@
-// import React from "react";
-// import { render, screen } from "@testing-library/react";
-// import userEvent from "@testing-library/user-event";
-// import { Table } from "./Table";
-
-// describe("Table", () => {
-//   it("renders...", () => {});
-// });
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import { Table, TableProps } from "./Table";
+
+const testColumns: TableProps["columns"] = [
+  { property: "name", label: "Name" },
+  { property: "email", label: "Email" },
+];
+
+const testData: TableProps["data"] = [
+  { id: 1, name: "John Doe", email: "john.doe@example.com" },
+  { id: 2, name: "Jane Doe", email: "jane.doe@example.com" },
+];
+
+describe("Table", () => {
+  const renderComponent = (
```

Comment:
I think you should have just used the `defaultProps` and `additionalProps` pattern here, would have made your tests more consistent and easier to work with.